.github/workflows/helm.yml

```
@@ -39,38 +35,58 @@ jobs:
     outputs:
       charts_path: ${{ steps.set-path.outputs.path }}
       repository_name: ${{ steps.set-path.outputs.repo }}
-  lint:
-    name: Lints
-    runs-on: ubuntu-latest
-    needs: prepare-charts-path
-    steps:
-        - uses: actions/checkout@v4
-        - uses: azure/setup-helm@v3.5
-          with:
-            version: "v3.12.0"
-          id: install
-        - name: Build dependencies
-          run: helm dep update ${{ needs.prepare-charts-path.outputs.charts_path }}
-        - name: Lint Chart
-          run: helm lint ${{ needs.prepare-charts-path.outputs.charts_path }}
-  test-if-version-changed:
-    name: Tests if the Chart version is updated and CHANGELOG is updated for the version
+
+  tests:
+    name: Tests
     runs-on: ubuntu-latest
-    needs: prepare-charts-path
+    needs: set-path
     steps:
       - uses: actions/checkout@v4
+
       - uses: azure/setup-helm@v3.5
         with:
           version: "v3.12.0"
         id: install
 
-      - name: Install tools
-        run: sudo snap install yq
-      - name: Get Chart version
+      - name: Checkout bitnami-labs/readme-generator-for-helm
+        uses: actions/checkout@v4
+        with:
+          repository: "bitnami-labs/readme-generator-for-helm"
+          ref: "2.5.0"
+          path: readme-generator-for-helm
+
+      - name: Install dependencies
+        run: sudo snap install yq && cd readme-generator-for-helm && npm install && cd ..
+
+      - name: Helm repo add
+        run: |
+          REPOS_JSON="${{ inputs.helm-repositories }}"
```

```suggestion
          REPOS_JSON='${{ inputs.helm-repositories }}'
```

It is the fix I pushed the other day

---

Indeed, copy-pasted before rebasing. Sorry for that, thanks for spotting it!